### PR TITLE
Socket timeout fix

### PR DESCRIPTION
**Pull request by 0lvin**

### CODE REVIEW

File: cloudify_terminal/tasks.py [hunk 1]
```diff
@@ -74,14 +78,20 @@ def run(**kwargs):
...
-            ctx.logger.info("Can't connect to %s with %s" % (
-                repr(ip), str(ex)
-            ))
+            ctx.logger.info("Can't connect to:{} with exception:{} and type:{}"
+                            .format(repr(ip), str(ex), str(type(ex))))
     else:
-        raise cfy_exc.NonRecoverableError(
-            "please check your ip list"
-        )
+        raise cfy_exc.OperationRetry(message="Please check your ip list")
```
Comment (astianseb) [inline cloudify_terminal/tasks.py]: I do not like this message. It's bit confusing provided that we're sending logger before. Suggest to remove it.

File: cloudify_terminal/tasks.py [hunk 2]
```diff
@@ -74,14 +78,20 @@ def run(**kwargs):
...
+                            .format(str(err)))
+
         except Exception as ex:
-            ctx.logger.info("Can't connect to %s with %s" % (
-                repr(ip), str(ex)
-            ))
+            ctx.logger.info("Can't connect to:{} with exception:{} and type:{}"
+                            .format(repr(ip), str(ex), str(type(ex))))
     else:
-        raise cfy_exc.NonRecoverableError(
```
Comment (astianseb) [inline cloudify_terminal/tasks.py]: I do not like that we're removing NonRecoverable error. We may spot some errors which are not captured properly and deployment may get stuck. I prefer to leave NonRecoverableError as a safety net ....but we need to properly report which error and type caused it.

File: cloudify_terminal/tasks.py (hunk 1 above)
Comment (0lvin) [reply cloudify_terminal/tasks.py]: We can replace it to something more userfriendly, maybe "let's try one more time?"

File: cloudify_terminal/tasks.py (hunk 2 above)
Comment (0lvin) [reply cloudify_terminal/tasks.py]: We need in someway separate "recoverable" errors, like network link will be restored until next retry and case that we never will have ip (from list) with alive connection. So we can think "all troubles is temporary, lets try one more time"  for now.

Comment (astianseb) [reply cloudify_terminal/tasks.py]: It may work :) Let me test it.

**Pull request merged**